images/rootfs.yml.in

```diff
@@ -38,9 +38,8 @@ services:
    - name: pillar
      image: PILLAR_TAG
 files:
-   - path: /containers/services/pillar/lower/opt/zededa/bin/versioninfo
+   - path: /etc/eve-release
```

Comment:
baseosmgr will still look for the file in the old path, and crash when it doesn't find it.
Path of least resistance might be to install the file in both places for now; changing the location is painful as we've seen with going from pre-eve to eve and back again.

Reply:
That's not an issue since a symlink now exists in the pillar container pointing to /hostsfs/etc/eve-release